src/components/core/InteractiveLink/InteractiveLink.tsx

```diff
@@ -61,17 +70,25 @@ export default function InteractiveLink({ to, href, children, className, onClick
     );
   }
 
+  if (onClick) {
```

Comment:
Added this block since there is a scenario that we want to retain the same link design but with `onClick` without any link